### DISCUSSION TITLE
How to center an image with a :scale width

### DISCUSSION THREAD

**Discussion by LouisDeconinck** [andymeneely/squib#356] [Q&A] [answered]
You can use :middle or :center to find the middle x and y coordinates for your card. However, if you place an image there, the image will start there and not be centered there. Normally you could subtract half of the images' width or height from the middle of the card, however, if you use :scale for the width or height of the image you don't know this value. 

How to center an image with a :scale width?

**Comment by andymeneely** [comment, marked as answer]
At the moment we don't have that. I believe I documented ideas for it in #238. Would be easier to do today than it was in the past.

**Reply by LouisDeconinck** [reply]
I assume centering text on your card also does not yet exist?
